### PR TITLE
ROX-17337: let 4.13 nightlies run on stackrox cluster pool

### PR DESCRIPTION
**Pull request by tommartensen**
/hold

### CODE REVIEW

**Comment by tommartensen** [comment]
/pj-rehearse pull-ci-stackrox-stackrox-tm-cluster-pool-ocp-4-13-qa-e2e-tests

**Comment by tommartensen** [comment]
/pj-rehearse pull-ci-stackrox-stackrox-tm-cluster-pool-ocp-4-13-qa-e2e-tests

**Comment by tommartensen** [comment]
/pj-rehearse pull-ci-stackrox-stackrox-tm-cluster-pool-ocp-4-13-qa-e2e-tests

**Comment by tommartensen** [comment]
/pj-rehearse pull-ci-stackrox-stackrox-tm-cluster-pool-ocp-4-13-qa-e2e-tests

**Comment by tommartensen** [comment]
/pj-rehearse pull-ci-stackrox-stackrox-nightlies-ocp-4-13-fips-qa-e2e-tests

**Comment by tommartensen** [comment]
> fips not enabled in node ip-10-0-131-70.us-west-2.compute.internal but should be, exiting

should we keep the fips test on the old IPI setup or investigate enabling FIPS mode in cluster pool?

File: ci-operator/config/stackrox/stackrox/stackrox-stackrox-nightlies__ocp-4-13.yaml [hunk 1]
```diff
@@ -21,68 +21,97 @@ resources:
 test_binary_build_commands: .openshift-ci/dispatch.sh test-binary-build-commands
 tests:
 - as: qa-e2e-tests
-  postsubmit: true
```
Comment (gavin-stackrox) [inline ci-operator/config/stackrox/stackrox/stackrox-stackrox-nightlies__ocp-4-13.yaml:24]: These need to stay as postsubmit.

**Review by gavin-stackrox — CHANGES_REQUESTED** [review]

File: ci-operator/config/stackrox/stackrox/stackrox-stackrox-nightlies__ocp-4-13.yaml (hunk 1 above)
Comment (tommartensen) [reply ci-operator/config/stackrox/stackrox/stackrox-stackrox-nightlies__ocp-4-13.yaml:24]: Needed to have it as presubmit for rehearsing. Now reverted.

**Pull request merged**